### ISSUE TITLE
julia 1.3.0 incompatibility

### ISSUE THREAD

**Issue by johnnychen94** [JuliaTime/TimeZones.jl#277]
The following is a CI log from a failed CompatHelper job https://github.com/JuliaImages/ImageQualityIndexes.jl/runs/879908860?check_suite_focus=true

```julia
2020-07-17T00:05:19.4961503Z   Building TimeZones → `~/.julia/packages/TimeZones/9Cv9B/deps/build.log`
2020-07-17T00:05:23.4477905Z ┌ Error: Error building `TimeZones`: 
2020-07-17T00:05:23.4479311Z │ ERROR: LoadError: LoadError: UndefVarError: do_artifact_str not defined
2020-07-17T00:05:23.4480272Z │ Stacktrace:
2020-07-17T00:05:23.4481036Z │  [1] top-level scope at /home/runner/.julia/packages/TimeZones/9Cv9B/src/compat.jl:4
2020-07-17T00:05:23.4481736Z │  [2] include at ./boot.jl:328 [inlined]
2020-07-17T00:05:23.4482859Z │  [3] include_relative(::Module, ::String) at ./loading.jl:1105
2020-07-17T00:05:23.4483696Z │  [4] include at ./Base.jl:31 [inlined]
2020-07-17T00:05:23.4484971Z │  [5] include(::String) at /home/runner/.julia/packages/TimeZones/9Cv9B/src/TimeZones.jl:1
2020-07-17T00:05:23.4486098Z │  [6] top-level scope at /home/runner/.julia/packages/TimeZones/9Cv9B/src/TimeZones.jl:49
2020-07-17T00:05:23.4486828Z │  [7] include at ./boot.jl:328 [inlined]
2020-07-17T00:05:23.4487873Z │  [8] include_relative(::Module, ::String) at ./loading.jl:1105
2020-07-17T00:05:23.4491768Z │  [9] include(::Module, ::String) at ./Base.jl:31
2020-07-17T00:05:23.4492177Z │  [10] top-level scope at none:2
2020-07-17T00:05:23.4492524Z │  [11] eval at ./boot.jl:330 [inlined]
2020-07-17T00:05:23.4492991Z │  [12] eval(::Expr) at ./client.jl:425
2020-07-17T00:05:23.4493354Z │  [13] top-level scope at ./none:3
2020-07-17T00:05:23.4493736Z │ in expression starting at /home/runner/.julia/packages/TimeZones/9Cv9B/src/compat.jl:1
2020-07-17T00:05:23.4494141Z │ in expression starting at /home/runner/.julia/packages/TimeZones/9Cv9B/src/TimeZones.jl:49
2020-07-17T00:05:23.4494817Z │ ERROR: LoadError: Failed to precompile TimeZones [f269a46b-ccf7-5d73-abea-4c690281aa53] to /home/runner/.julia/compiled/v1.3/TimeZones/2uWP2_XvTLz.ji.
2020-07-17T00:05:23.4495185Z │ Stacktrace:
2020-07-17T00:05:23.4495706Z │  [1] error(::String) at ./error.jl:33
2020-07-17T00:05:23.4496299Z │  [2] compilecache(::Base.PkgId, ::String) at ./loading.jl:1283
2020-07-17T00:05:23.4496844Z │  [3] _require(::Base.PkgId) at ./loading.jl:1024
2020-07-17T00:05:23.4497402Z │  [4] require(::Base.PkgId) at ./loading.jl:922
2020-07-17T00:05:23.4497884Z │  [5] require(::Module, ::Symbol) at ./loading.jl:917
2020-07-17T00:05:23.4498225Z │  [6] include at ./boot.jl:328 [inlined]
2020-07-17T00:05:23.4498768Z │  [7] include_relative(::Module, ::String) at ./loading.jl:1105
2020-07-17T00:05:23.4499358Z │  [8] include(::Module, ::String) at ./Base.jl:31
2020-07-17T00:05:23.4499873Z │  [9] include(::String) at ./client.jl:424
2020-07-17T00:05:23.4500198Z │  [10] top-level scope at none:5
2020-07-17T00:05:23.4501096Z │ in expression starting at /home/runner/.julia/packages/TimeZones/9Cv9B/deps/build.jl:1
2020-07-17T00:05:23.4501634Z └ @ Pkg.Operations /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.3/Pkg/src/backwards_compatible_isolation.jl:649
```

**Comment by omus** [comment]
Thanks for the report. I'll investigate shortly

**Comment by omus** [comment]
Issue only seems to affect Julia 1.3.0. Interestingly the issue I was addressing on 1.3.1 isn't present in 1.3.0 so the fix is pretty easy. I'll verify the change locally first.

**Comment by omus** [comment]
Addressed in TimeZones 1.3.1
